src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Services/GreeterService.cs

```diff
@@ -1,16 +1,9 @@
 using Grpc.Core;
-using GrpcService_CSharp;
 
 namespace GrpcService_CSharp.Services;
 
 public class GreeterService : Greeter.GreeterBase
```

Comment:
Nit: Although the `ILogger<GreeterService>` was unused before perhaps we can add a log to the `SayHello` method to more fully demonstrate primary constructors here?